card/mod/core/set/all/collection.rb

```diff
@@ -228,10 +228,11 @@ def edit_fields
   end
 
   def normalized_edit_fields
-    edit_fields.map do |name, options|
-      options ||= Card.quick_fetch(name).name
+    edit_fields.map do |name_or_card, options|
+      next [name_or_card, options || {}] if name_or_card.is_a?(Card)
```

Comment:
And, looking in the code, I don't see where edit_structure ever contains cards?  Does this really fix something?